### PR TITLE
Empty view

### PR DESCRIPTION
**Pull request by mattprecious**
screenshot-2015-04-03_01 14 42 136

### CODE REVIEW

File: src/main/res/layout/trending_view.xml [hunk 1]
```diff
@@ -107,6 +107,39 @@
           android:text="@string/error_message"
           />
     </RelativeLayout>
+      <RelativeLayout
```
Comment (JakeWharton) [inline src/main/res/layout/trending_view.xml]: Indent is off

**Comment by JakeWharton** [comment]
lgtm

**Comment by JakeWharton** [comment]
:neutral_face:

File: src/main/res/layout/trending_view.xml (hunk 1 above)
Comment (mattprecious) [inline src/main/res/layout/trending_view.xml]: :neutral_face:

**Pull request merged**